packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -90,5 +90,8 @@ export const messages = {
   force_alias_list_text: 'SFDX: List All Aliases',
   force_org_display_default_text:
     'SFDX: Display Org Details for Default Scratch Org',
-  force_org_display_username_text: 'SFDX: Display Org Details...'
+  force_org_display_username_text: 'SFDX: Display Org Details...',
+  force_debugger_query_session_text: 'query for Apex Debugger session',
+  force_debugger_stop_text: 'SFDX: Stop Apex Debugger Session',
+  force_debugger_stop_none_found_text: 'No Apex Debugger session found.'
 };
```

Comment:
The debugger session stop command in the command palette is a two-step process. First, we query for active session. If there is none, we show this message:

![stopsessioncommandnoop](https://user-images.githubusercontent.com/7286437/30933702-29fdcfa6-a380-11e7-9b39-fae49f917cae.png)

If there is an active session, we update its status to stop it.

![stopsessioncommandsuccess](https://user-images.githubusercontent.com/7286437/30933752-50ab00c4-a380-11e7-82c5-ad880843c56e.png)